packages/libchromaprint/build.sh

```diff
@@ -5,14 +5,16 @@ TERMUX_PKG_MAINTAINER="@termux"
 _COMMIT=aa67c95b9e486884a6d3ee8b0c91207d8c2b0551
 _COMMIT_DATE=20221217
 TERMUX_PKG_VERSION=1.5.1-p${_COMMIT_DATE}
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL=git+https://github.com/acoustid/chromaprint
 TERMUX_PKG_SHA256=5a880f6e976fdbbfbc1d5487d27cf59fba7398c675c6cb5069aaf3d3cff716a7
 TERMUX_PKG_GIT_BRANCH=master
-TERMUX_PKG_DEPENDS="ffmpeg, libc++"
+TERMUX_PKG_DEPENDS="fftw, libc++"
+TERMUX_PKG_BUILD_DEPENDS="ffmpeg"
```

Comment:
```suggestion
TERMUX_PKG_BUILD_DEPENDS="ffmpeg"
TERMUX_PKG_BREAKS="chromaprint (<< 1.5.1-p20221217-2)"
TERMUX_PKG_REPLACES="chromaprint (<< 1.5.1-p20221217-2)"
```
When renaming/spliting/replacing a package, you need to specify these 2 variables so that dpkg can handle the package update.

Reply:
Shouldn't it be (<< 1.5.1-p20221217-3) or am I missing something

Reply:
My bad. You are correct.